app/services/articles/feeds/configs/falcon_thunder.rb

```diff
@@ -0,0 +1,166 @@
+module Articles
+  module Feeds
+    module Configs
+      class FalconThunder
```

Comment:
I used names like `FalconThunder` to illustrate the open question of how might we name and describe the feeds.

Don't pay too much attention to this yet, but if you have opinions, they are welcome.